File: lista1_js/13.valor_real.js
//13. Leia um valor em real (R$), calcule e escreva 70% deste valor.
import { question } from "readline-sync"

//Entrada
const valor = Number(question('Informe um valor em reais: R$'))

//Processamento
const novo_valor = valor * (70/100)

//Saída
console.log(`\n>>> 70% de R$${valor.toFixed(2)} é R$${novo_valor.toFixed(2)}.`)

File: lista1_py/13.valor_real.py
#13. Leia um valor em real (R$), calcule e escreva 70% deste valor.

#Entrada
valor = float(input('Informe um valor em reais: R$'))

#Processamento
novo_valor = valor * (70/100)

#Saída
print(f'\n>>> 70% de R${valor:.2f} é R${novo_valor:.2f}.')